spec/src/main/resources/META-INF/smithy/bsp/bsp.smithy

```diff
@@ -508,7 +508,11 @@ operation BuildTargetCleanCache {
 
 
 /// Represents the identifier of a BSP request.
-string RequestId
+@untaggedUnion
+union RequestId {
+    string: String
+    integer: Integer
+}
```

Comment:
instead of this breaking change I would suggest to rename `RequestId` to `OriginId` (as it is a concept introduced by BSP) and use `RequestId` only in `CancelParams`. This way `RequestId` would be the JsonRPC id